### PR TITLE
Us13 15 model manager bd

### PR DESCRIPTION
**Pull request by Vannou28**

### CODE REVIEW

**Review by RaphaelBS-WCS — APPROVED** [review]
Good Work

**Review by Erdnaxela45 — APPROVED** [review]

File: src/Controller/TestimonyController.php [hunk 1]
```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Controller;
+
+use App\Model\TestimonyManager;
+
+class TestimonyController extends AbstractController
+{
+    /**
+     * List Testimonys
```
Comment (LucyMerlier) [inline src/Controller/TestimonyController.php:10]: singular : testimony => plural : testimonIEs ;)

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment + there is a whole CRUD in your controller, which are 5 different user stories ;)

**Comment by RaphaelBS-WCS** [comment]
replace by others

**Pull request closed without merging**